includes/rest-api/Controllers/Version2/class-wc-rest-system-status-v2-controller.php

```diff
@@ -1220,6 +1227,14 @@ public function get_pages() {
 				}
 			}
 
+			// Block checks.
+			if ( $values['block'] && get_post( $page_id ) ) {
+				$block_required = true;
+				if ( WC_Blocks_Utils::does_page_contains_block( $page_id, $values['block'] ) ) {
+					$block_present = true;
+				}
```

Comment:
Can probably just do something like:

```suggestion
				$block_present = WC_Block_Utils:does_page_contains_block( $page_id, $values['block'] );
```